data/campaigns/Sceptre_of_Fire/scenarios/4_Gathering_Materials.cfg

```diff
@@ -323,6 +324,11 @@
         # wmllint: recognize Alanin
         # wmllint: recognize Krawg
 
+        [music]
+            name=into_the_shadows.ogg
```

Comment:
Somehow I feel like into the shadows doesn't fit well here. If you don't like suspense, then maybe nunc dimittis?

Reply:
I'll give that a try, thanks.